vocabulary.md

```diff
@@ -9,19 +9,21 @@
-- **Conversation Stream** - Each Conversational Endpoint generates one or more Conversation Streams based upon the capabilities of the Endpoint and the preferences of the Agent.  A Conversation Stream is associated with a particular Agent and may include any media type including text, audio, video, and application UI events.  
-- **Conversational Agent** - A digital participant in a conversation.   This may be an application with a consistent persona, such as Amazon Alexa, Google Assistant, the Target Google Assistant Action, a Facebook Messenger chatbot, or an IVR system at a bank.  Conversational agent is a common term utilized in the area of Dialogue System research and university level instruction; the term often is used to describe a human participant in a conversation.  For clarity of reference, however, the OVN will use the term "user" to identify a human participant.  (See "user" below.)
+- **Conversation** - a collaborative linguistic act between two or more conversational agents to achieve a user goal.
+- **Conversational Access Point** - the means of carrying a signal to a conversational platform.  Users initiate and conduct conversationds through _access points_.  An access point can be physical or virtual.  Current examples of conversational access points iclude smartphones, smart speakers, personal compiuters, wearables, or microphone-enabled smart home hubs. Access points may be referred to elsewhere as a "device" or a "channel."  _In the interoperable world, conversational access points will enable connection to any conversational platform and agent_.
+- **Conversational Agent** - A digital participant in a conversation.  Conversational agent is a common term utilized in the area of Dialogue System research and university level instruction; the term also used to describe a human participant in a conversation.  For clarity of reference, however, the Open Voice Network will use the term "user" to identify a human participant.  (See "user" below.)  Within the OVON Interoperability Initiative, a _conversational agent_ is perceived by users to be a single conversational actor, and with the ability to not only provide requested information, but _operate on behalf of the user_.  It uses the infrastructure of a conversational platform, and uses one or more conversational capabilities to hold conversations with the user, with continuity of knowledge and persona, and a name by which it can be addressed.  Enterprise conversational agents reach outside the corporate firewall to fulfill user intents.  _In the interoperable world, there will be both general-purpose and specific-purpose conversational agents.  Dialogs, data, context, and controls will be passed between agents according to standardized protocols. 
```

Comment:
I would like to see a reference to this research/instruction to shore up our vocabulary.